packages/keepalived/packaging

```diff
@@ -0,0 +1,12 @@
+# abort script on any command that exits with a non zero value
+set -e -x
+
+tar xzvf keepalived/keepalived-1.2.24.tar.gz
```

Comment:
should probably indicate URL to sources in order to ease future upgrades to this releases (preferably HTTPS url and SHA/MD5)